lib/entities/content-type-fields.ts

```diff
@@ -15,11 +16,23 @@ interface RegExp {
   flags: string
 }
 
+interface NodesValidation {
+  [BLOCKS.EMBEDDED_ENTRY]?: Pick<ContentTypeFieldValidation, 'size' | 'linkContentType'>[]
+  [INLINES.EMBEDDED_ENTRY]?: Pick<ContentTypeFieldValidation, 'size' | 'linkContentType'>[]
+  [INLINES.ENTRY_HYPERLINK]?: Pick<ContentTypeFieldValidation, 'size' | 'linkContentType'>[]
+  [BLOCKS.EMBEDDED_ASSET]?: Pick<ContentTypeFieldValidation, 'size'>[]
+  [INLINES.ASSET_HYPERLINK]?: Pick<ContentTypeFieldValidation, 'size'>[]
+  [BLOCKS.EMBEDDED_RESOURCE]?: {
+    validations: Pick<ContentTypeFieldValidation, 'size'>[]
+    allowedResources: ContentTypeAllowedResources
+  }
+}
+
```

Comment:
I'd say we're missing the `message` property in some of these

Reply:
Yes we are